### PR TITLE
Fix how tps benchmark is calculated

### PR DESCRIPTION
**Pull request by miiu96**
Modify how tps benchmark is calculated. 
Now all the peer transactions and smart contract results are ignored on computation for all the metrics.

### CODE REVIEW

File: core/statistics/tpsBenchmark.go [hunk 1]
```diff
@@ -322,6 +327,31 @@ func (s *TpsBenchmark) updateStatistics(header *block.MetaBlock) error {
...
+	for idx := 0; idx < len(metaBlock.ShardInfo); idx++ {
+		totalTxs += getNumTxsFromMiniblocksWithoutPeerTxs(metaBlock.ShardInfo[idx].ShardMiniBlockHeaders)
+	}
+
+	return totalTxs
+}
+
+func getNumTxsFromMiniblocksWithoutPeerTxs(miniblocks []block.MiniBlockHeader) uint64 {
+	totalTxs := uint64(0)
+	for idx := 0; idx < len(miniblocks); idx++ {
```
Comment (AdoAdoAdo) [inline core/statistics/tpsBenchmark.go:344]: can you use range instead?

File: core/statistics/tpsBenchmark.go [hunk 2]
```diff
@@ -322,6 +327,31 @@ func (s *TpsBenchmark) updateStatistics(header *block.MetaBlock) error {
 	return nil
 }
 
+func getNumOfTxsWithoutPeerTxs(metaBlock *block.MetaBlock) uint64 {
+	// get number of transactions from metablock miniblocks
+	totalTxs := getNumTxsFromMiniblocksWithoutPeerTxs(metaBlock.MiniBlockHeaders)
+
+	// get number of transactions from shard blocks that are included in metablock
+	for idx := 0; idx < len(metaBlock.ShardInfo); idx++ {
```
Comment (AdoAdoAdo) [inline core/statistics/tpsBenchmark.go:335]: can you use range instead?

File: core/statistics/tpsBenchmark.go (hunk 1 above)
Comment (miiu96) [reply core/statistics/tpsBenchmark.go:344]: done.

File: core/statistics/tpsBenchmark.go (hunk 2 above)
Comment (miiu96) [reply core/statistics/tpsBenchmark.go:335]: done.

**Review by sasurobert — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
System tests passed.

**Pull request merged**